docs/Using-Fleet/Adding-hosts.md

```diff
@@ -257,10 +260,10 @@ Osqueryd will inherit the privileges from Orbit and does not need explicit permi
 #### Creating the profile
 Depending on your MDM, this might be possible in the UI or require a custom profile. If your MDM has a feature to configure *Policy Preferences*, follow these steps:
 
-1. Configure the identifier type to “path”
+1. Configure the identifier type to “path.”
 2. Paste the full path to Orbit as the identifier.
-3. Paste the full code signing identifier into the code requirement field.
-4. Allow “Access all files”. Access to Downloads, Documents etc is inherited from this.
+3. Paste the full code signing identifier into the code requirement field. 
```

Comment:
Should "code requirement" be capitalized or bold?

Reply:
I think these instructions are a walk through of the SimpleMDM UI (this is the MDM Fleet uses).

@GuillaumeRoss if possible can you please paste a screenshot of the *Policy preferences* feature in Simple MDM UI?

This way, we can find out if "code requirement" is the title of a field in the SimpleMDM UI.

Reply:
@GuillaumeRoss, I just wanted to tag you as a reminder for this.